tests/integration/select2-methods.js

```diff
@@ -1,3 +1,4 @@
+/* globals start, stop, setTimeout */
```

Comment:
Sorry, but apparently we don't have [assert.async()](https://api.qunitjs.com/async/) in our version of QUnit !